CONTRIBUTING.md

```diff
@@ -2,6 +2,18 @@
 
 Thanks for your interest in contributing to OpenLayers 3.
 
+## Asking Questions
+
+Please ask questions about using the library and its API on [stackoverflow using the tag 'openlayers-3'](http://stackoverflow.com/questions/tagged/openlayers-3).
```

Comment:
I am wondering what "and its API" brings. To me, StackOverflow makes sense for questions about using the library. Everything else can go to the mailing list.